BERTopicModel.py

```diff
@@ -0,0 +1,49 @@
+from ClusteringModel import ClusteringModel
+from bertopic import BERTopic
+
+
+class BERTopicModel(ClusteringModel, BERTopic):
+    def __init__(self, language='english', top_n_words=10, n_gram_range=(1, 1), min_topic_size=10, nr_topics=None,
+                 low_memory=False, calculate_probabilities=False, seed_topic_list=None, zeroshot_topic_list=None,
+                 zeroshot_min_similarity=0.7, embedding_model=None, umap_model=None, hdbscan_model=None,
+                 vectorizer_model=None, ctfidf_model=None, representation_model=None, verbose=False):
+        self.language = language
+        self.top_n_words = top_n_words
+        self.n_gram_range = n_gram_range
+        self.min_topic_size = min_topic_size
+        self.nr_topics = nr_topics
+        self.low_memory = low_memory
+        self.calculate_probabilities = calculate_probabilities
+        self.seed_topic_list = seed_topic_list
+        self.zeroshot_topic_list = zeroshot_topic_list
+        self.zeroshot_min_similarity = zeroshot_min_similarity
+        self.embedding_model = embedding_model
+        self.umap_model = umap_model
+        self.hdbscan_model = hdbscan_model
+        self.vectorizer_model = vectorizer_model
+        self.ctfidf_model = ctfidf_model
+        self.representation_model = representation_model
+        self.verbose = verbose
+
+        super().__init__(language=language, top_n_words=top_n_words,
+                         n_gram_range=n_gram_range, min_topic_size=min_topic_size,
+                         nr_topics=nr_topics, low_memory=low_memory,
+                         calculate_probabilities=calculate_probabilities,
+                         seed_topic_list=seed_topic_list,
+                         zeroshot_topic_list=zeroshot_topic_list,
+                         zeroshot_min_similarity=zeroshot_min_similarity,
+                         embedding_model=embedding_model, umap_model=umap_model,
+                         hdbscan_model=hdbscan_model,
+                         vectorizer_model=vectorizer_model, ctfidf_model=ctfidf_model,
+                         representation_model=representation_model, verbose=verbose)
+
+    def getClusters(self, documents):
```

Comment:
Consider using `override` from the `typing` module to indicate that the method is derived.